tests/playwright/assets/breakpoints.ts

```diff
@@ -36,7 +36,7 @@ export default class {
 			await this.page.goto( `/wp-admin/post.php?post=${ experimentPostId }&action=elementor` );
 		} else {
 			await this.page.reload();
-
+			await this.page.reload();
```

Comment:
double page reload?

Reply:
yes. After the first one, left panel is hanging. Looks like second one should help.